docs/code_ref/coordinates.rst

```diff
@@ -3,14 +3,87 @@
 SunPy coordinates
 =================
 
-The SunPy coordinates submodule is an implementation of the common solar physics
-coordinate frames using the :ref:`Astropy coordinates framework <astropy:astropy-coordinates>`.
+This sub-package contains:
+
+* A robust framework for working with solar-physics coordinate systems
+* Functions to obtain the locations of solar-system bodies (`sunpy.coordinates.ephemeris`)
+* Functions to calculate Sun-specific coordinate information (`sunpy.coordinates.sun`)
+
+The SunPy coordinate framework extends the
+:ref:`Astropy coordinates framework <astropy:astropy-coordinates>`.
+
+.. list-table:: Supported coordinate systems
```

Comment:
Our CSS isn't great for the heading of this table. Could I suggest you manually title this:
```suggestion
Supported coordinate systems
---------------------------------------

.. list-table:: 
```

Reply:
Done